docs/software/cloud.md

```diff
@@ -364,9 +373,35 @@ Basically, it allows you to create a complete system backup, using a simple web
     - Click `Save`.
     - Restart service with `systemctl restart urbackupsrv`.
 
+        ???+ note "Backup storage location directory and access rights"
+            It is necessary that the backup directory exists, UrBackup does not create the directory you set within the Backup Storage Path value.  
+            Additionally, the access rights resp. ownership needs to be set appropriately. It can be achieved via
+
+            ```sh
+            mkdir </path/to/backup/storage>
+            chmod -R urbackup:urbackup </path/to/backup/storage>
```

Comment:
To avoid users typing those braces literally:
```suggestion
            mkdir /path/to/backup/storage
            chmod -R urbackup:urbackup /path/to/backup/storage
```